### PR TITLE
Creating HA-Google-Home json

### PR DESCRIPTION
**Pull request by leikoilja**

### CODE REVIEW

**Comment by leikoilja** [comment]
I'm confused... failed action says "Package uuid is not compatible with Python 3.8 standard library", but `uuid` is build-in library in python 3.8 and is indeed present in `std_libs`. 
I think there is a bug in the validation method, addressed here - https://github.com/home-assistant/wheels-custom-integrations/pull/314

File: requirements.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
         "cups-dev",
         "eudev-dev",
         "ffmpeg-dev",
+        "g++",
```
Comment (frenck) [inline requirements.json:11]: `g++` is already part of `build-base`, see:
<https://pkgs.alpinelinux.org/package/v3.13/main/x86_64/build-base>

**Review by frenck — CHANGES_REQUESTED** [review]

File: requirements.json (hunk 1 above)
Comment (leikoilja) [reply requirements.json]: Oh, that's great!

**Review by frenck — APPROVED** [review]
Thanks, @leikoilja 👍

**Pull request merged**